app/views/miq_request/_prov_field.html.haml

```diff
@@ -48,6 +48,16 @@
     .form-group
       %label.col-md-2.control-label{:valign => "top"}
         = field_hash[:description]
+        - if field_hash[:help]
+          %a{:tabindex => 0,
+             :role             => "button",
+             :"data-toggle"    => "popover",
+             :"data-trigger"   => "focus",
+             :"data-html"      => "true",
+             :title            => "",
+             :"data-content"   => "#{field_hash[:help]}",
+             :"data-placement" => "top"}
+            %span.fa.fa-info-circle
```

Comment:
If that's all, that's awesome. I was worried that, because of the way the html is loaded, the selector wouldn't be able to find the element in the dom (since it's not mounted at the point of execution yet? I'm actually unsure of this; the only thing I know is that after jquery's `html` method is called the elements are mounted and can be selected).

I'll throw this in, test it a bit and if it's working repost.  Thanks!!